site/docs/v1/tech/apis/_user-registration-combined-request-body.adoc

```diff
@@ -2,6 +2,8 @@ This request requires that you specify both the User object and the User Registr
 
 ==== Request Body
 
+:set_password_field_config_location: Tenant or Application
```

Comment:
Since this is just a replacement value, will it be clear enough that tenant is the "default" value, and it can optionally be overridden by the application config, if available?

Reply:
Thanks, clarified ths.